infrastructure/core/locals.tf

```diff
@@ -1,3 +1,17 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
```

Comment:
Do we need to have a linter for those as well? 
(Not suggesting for this PR, just wondering)